internal/notification/pubsub/pubsub.go

```diff
@@ -97,10 +103,19 @@ func (ps PubSub) Publish(ctx context.Context, msg Message) error {
 	ps.topic.Publish(ctx, &cloudpubsub.Message{
 		Data: data,
 	})
-	logger.WithField("size", len(data)).Debug("published data to Pub/Sub")
+	logger.WithField("size", len(data)).Debug("event published to Pub/Sub")
 	return nil
 }
 
+// Stop is a wrapper around Cloud Run Pub/Sub package's Stop method on Topic.
+//
+// It sends all remaining published messages and stop goroutines created for
+// handling publishing. Returns once all outstanding messages have been sent or
+// have failed to be sent.
+func (ps PubSub) Stop() {
+	ps.topic.Stop()
+}
```

Comment:
I was imagining this would be returned as a `func()` from `New()`. It would be harder to ignore or miss that way.

Reply:
That's also possible (and asked about in my previous comment). Let's see how this looks in the next PR as we implement the package in the rollout